### ISSUE TITLE
colorid not checked in dependency checking scripts

### ISSUE THREAD

**Issue by kapsakcj** [lskatz/SneakerNet#27]
**Describe the bug**
checkdeps scripts do not check for colorid

**To Reproduce**
I have this bit in my dockerfile to check for all dependencies
```Dockerfile
RUN ./SneakerNet-0.8.14/scripts/SneakerNet.checkdeps.pl default && \
 ./SneakerNet-0.8.14/scripts/SneakerNet.checkdeps.pl metagenomics && \
 ./SneakerNet-0.8.14/scripts/SneakerNet.checkdeps.pl cryptosporidium && \
 ./SneakerNet-0.8.14/scripts/SneakerNet.checkdeps.pl iontorrent
```

**Expected behavior**
Expected to see `colorid 0.1.4.2` somewhere in the output from each of these (not sure which workflow/plugin executes colorid)

**Desktop (please complete the following information):**
 - OS: Ubuntu:xenial base OS in docker image
 - Version: using SN v0.8.14 release

**Comment by kapsakcj** [comment]
additionally missing checks for SalmID and chewBBACA

**Comment by lskatz** [comment]
Could you copy the newest plugins.conf and try again?  I have a feeling it's an older conf and just not part of the default workflow.

    cp -v config.bak/plugins.conf config/plugins.conv

**Comment by kapsakcj** [comment]
Tried that, but I don't think that fixed it. SalmID, ColorID, and chewBBACA aren't tested when running `SneakerNet.checkdeps.pl default` or `SneakerNet.checkdeps.pl cryptosporidium`
```bash
# these were run from inside the newly built container w/ SalmID, ColorID, and chewBBACA installed
# I cloned SneakerNet repo from the master/HEAD to get your latest commits

root@58c4964c6284:/SneakerNet# cp -v config.bak/plugins.conf config/plugins.conf
'config.bak/plugins.conf' -> 'config/plugins.conf'

root@58c4964c6284:/SneakerNet# ./scripts/SneakerNet.checkdeps.pl default
sn_parseSampleSheet.pl: sn_parseSampleSheet.pl: 1.1
addReadMetrics.pl: addReadMetrics.pl: 1.3
assembleAll.pl: assembleAll.pl: 1.3
assembleAll.pl: prodigal: Prodigal V2.6.2: February, 2015
assembleAll.pl: skesa: SKESA v.2.3.0
sn_mlst.pl: sn_mlst.pl: 1.1
sn_mlst.pl: mlst: mlst 2.16.2
sn_mlst.pl: rm: rm (GNU coreutils) 8.25
sn_detectContamination-kraken.pl: sn_detectContamination-kraken.pl: 3.0
sn_detectContamination-kraken.pl: kraken: Kraken version 1.1.1
sn_detectContamination-kraken.pl: zip: This is Zip 3.0 (July 5th 2008), by Info-ZIP.
baseBalance.pl: baseBalance.pl: 1.4
baseBalance.pl: cp: cp (GNU coreutils) 8.25
sn_staramr.pl: sn_staramr.pl: 1.4
sn_staramr.pl: staramr: staramr 0.5.1
sn_passfail.pl: sn_passfail.pl: 3.0
transferFilesToRemoteComputers.pl: transferFilesToRemoteComputers.pl: 1.6
transferFilesToRemoteComputers.pl: rsync: rsync  version 3.1.1  protocol version 31
transferFilesToRemoteComputers.pl: ssh: OpenSSH_7.2p2 Ubuntu-4ubuntu2.8, OpenSSL 1.0.2g  1 Mar 2016
sn_report.pl: sn_report.pl: 2.5
emailWhoever.pl: emailWhoever.pl: 2.2
emailWhoever.pl: sendmail: Version 8.15.2

DEPENDENCIES THAT WERE CHECKED:
blastn (BLAST+)
cp
kraken
kraken-report (Kraken)
kraken-translate (Kraken)
ktImportText (Krona)
mlst
prodigal
rm
rsync
run_assembly_filterContigs.pl (CG-Pipeline)
run_assembly_readMetrics.pl (CG-Pipeline)
run_prediction_metrics.pl (CG-Pipeline)
sendmail
skesa
ssh
staramr
zip


root@58c4964c6284:/SneakerNet# ./scripts/SneakerNet.checkdeps.pl cryptosporidium
sn_parseSampleSheet.pl: sn_parseSampleSheet.pl: 1.1
addReadMetrics.pl: addReadMetrics.pl: 1.3
sn_crypto_assembleAll.pl: sn_crypto_assembleAll.pl: 1.1
sn_crypto_assembleAll.pl: cat: cat (GNU coreutils) 8.25
sn_crypto_assembleAll.pl: head: head (GNU coreutils) 8.25
sn_crypto_assembleAll.pl: prodigal: Prodigal V2.6.2: February, 2015
sn_crypto_assembleAll.pl: shovill: shovill 1.0.4
sn_crypto_assembleAll.pl: sort: sort (GNU coreutils) 8.25
sn_crypto_assembleAll.pl: touch: touch (GNU coreutils) 8.25
sn_crypto_assembleAll.pl: uniq: uniq (GNU coreutils) 8.25
sn_mlst.pl: sn_mlst.pl: 1.1
sn_mlst.pl: mlst: mlst 2.16.2
sn_mlst.pl: rm: rm (GNU coreutils) 8.25
sn_detectContamination-kraken.pl: sn_detectContamination-kraken.pl: 3.0
sn_detectContamination-kraken.pl: kraken: Kraken version 1.1.1
sn_detectContamination-kraken.pl: zip: This is Zip 3.0 (July 5th 2008), by Info-ZIP.
sn_passfail.pl: sn_passfail.pl: 3.0
transferFilesToRemoteComputers.pl: transferFilesToRemoteComputers.pl: 1.6
transferFilesToRemoteComputers.pl: rsync: rsync  version 3.1.1  protocol version 31
transferFilesToRemoteComputers.pl: ssh: OpenSSH_7.2p2 Ubuntu-4ubuntu2.8, OpenSSL 1.0.2g  1 Mar 2016
emailWhoever.pl: emailWhoever.pl: 2.2
emailWhoever.pl: sendmail: Version 8.15.2

DEPENDENCIES THAT WERE CHECKED:
blastn (BLAST+)
cat
head
kraken
kraken-report (Kraken)
kraken-translate (Kraken)
ktImportText (Krona)
mlst
prodigal
rm
rsync
run_assembly_filterContigs.pl (CG-Pipeline)
run_assembly_metrics.pl (CG-Pipeline)
run_assembly_readMetrics.pl (CG-Pipeline)
run_prediction_metrics.pl (CG-Pipeline)
sendmail
shovill
sort
ssh
touch
uniq
zip

root@58c4964c6284:/SneakerNet# SalmID.py
usage: SalmID.py [-h] [-v] [-i your_fastqgz] [-e file_extension]
                 [-d directory] [-r percentage, coverage or taxonomy]
                 [-m quick or thorough]

SalmID - rapid Kmer based Salmonella identifier from sequence data

optional arguments:
  -h, --help            show this help message and exit
  -v, --version         show program's version number and exit
  -i your_fastqgz, --input_file your_fastqgz
                        Single fastq.gz file input, include path to file if
                        file is not in same directory
  -e file_extension, --extension file_extension
                        File extension, if specified without "--input_dir",
                        SalmID will attempt to ID all files with this
                        extension in current directory, otherwise files in
                        input directory
  -d directory, --input_dir directory
                        Directory which contains data for identification, when
                        not specified files in current directory will be
                        analyzed.
  -r percentage, coverage or taxonomy, --report percentage, coverage or taxonomy
                        Report either percentage ("percentage") of clade
                        specific kmers recovered, average kmer-coverage
                        ("cov"), or taxonomy (taxonomic species ID, plus
                        observed mean k-mer coverages and expected coverage).
  -m quick or thorough, --mode quick or thorough
                        Quick [quick] or thorough [thorough] mode

root@58c4964c6284:/SneakerNet# colorid
colorid 0.1.4.2
Henk C. den Bakker <henkcdenbakker@gmail.com>
BIGSI based taxonomic ID of sequence data

USAGE:
    colorid [SUBCOMMAND]

FLAGS:
    -h, --help       Prints help information
    -V, --version    Prints version information

SUBCOMMANDS:
    batch_id       classifies batch of samples reads
    build          builds a bigsi
    help           Prints this message or the help of the given subcommand(s)
    info           dumps index parameters and accessions
    read_filter    filters reads
    read_id        id's reads
    search         does a bigsi search on one or more fasta/fastq.gz files

root@58c4964c6284:/SneakerNet# chewBBACA.py

        USAGE : chewBBACA.py [module] -h

Select one of the following functions :

CreateSchema : Create a gene by gene schema based on genomes
AlleleCall : Perform allele call for target genomes
SchemaEvaluator : Tool that builds an html output to better navigate/visualize your schema
TestGenomeQuality : Analyze your allele call output to refine schemas
ExtractCgMLST : Select a subset of loci without missing data (to be used as PHYLOViZ input)
RemoveGenes : Remove a provided list of loci from your allele call output
PrepExternalSchema : Adapt an external schema to be used with chewBBACA.
JoinProfiles : join two profiles in a single profile file
UniprotFinder : get info about a schema created with chewBBACA
```

**Comment by lskatz** [comment]
ColorID has been added back into the default workflow.  Non-default workflow plugins can't be checked too easily.  wgMLST will not be added into default at this time.  Neither will SalmID at this time.  Changed in d540cc0.

I might make a dummy workflow to simply include all other plugins and so I will leave this ticket open.

**Comment by lskatz** [comment]
Fixed in v0.9.4 (7218958).  Added better dependency checking descriptions and added fake workflow that just has every plugin included in it.
